redfish-core/lib/hypervisor_system.hpp

```diff
@@ -443,20 +442,17 @@ inline void parseInterfaceData(
     ipv4StaticArray = nlohmann::json::array();
     for (auto& ipv4Config : ipv4Data)
     {
-        if (ipv4Config.isActive)
+        jsonResponse["InterfaceEnabled"] = ipv4Config.isActive;
```

Comment:
Isn't this just the last item in the for loop?

Reply:
Only one IP address is supported on VMI interface. So there will be always only one IP in this list

Reply:
Where do we check to ensure we only support 1 VMI interface?

Reply:
This logic lies in the networkd repo. So when a user configures ip on an interface, the existing dbus object will be deleted and a new dbus object will be created with user given inputs.

Ref: https://github.com/ibm-openbmc/phosphor-networkd/blob/1020/src/ibm/hypervisor-network-mgr-src/hyp_ethernet_interface.cpp#L570

Reply:
> This logic lies in the networkd repo. 

Yeah it seems like we should not just rely on the networkd repo, can we add a check here?

Reply:
Since, this PR is intended for another purpose, I will take it up in a different PR.
I will create a defect for myself for this and work on it once all my priority 1's are done.

Reply:
SW547380 is created